### ISSUE TITLE
Bookmarks!!!

### ISSUE THREAD

**Issue by JoshAtticus** [MystPi/ninetails#67]
So, I recently got Ninetails, and I love its design and simplicity, however I noticed one VERY IMPORTANT feature was missing, Bookmarks, I have like 274 of them in Firefox. It would also be nice to have a "Import from another browser" feature.

**Comment by webdev03** [comment]
I can try to make a bookmarks system. Where are the files for bookmarks stored in Firefox?

**Comment by hello-smile6** [comment]
> I can try to make a bookmarks system. Where are the files for bookmarks stored in Firefox?

Depends on the OS. :) On some OS'es, you don't stand a chance. But it's worth a try.

**Comment by webdev03** [comment]
Seems to be really hard to parse, maybe if Chromium imports bookmarks, check there?

**Comment by MystPi** [comment]
Bookmarks will probably be a feature in a future release.

**Comment by kccuber-scratch** [comment]
> I can try to make a bookmarks system. Where are the files for bookmarks stored in Firefox?

You can export bookmarks from firefox as JSON, that would be really easy

**Comment by webdev03** [comment]
Bookmarks were implemented in Catalyst.

**Comment by MystPi** [comment]
Implemented in v3.0.0.
